FreeRTOS_ARP.c

```diff
@@ -490,6 +490,11 @@ void vARPRefreshCacheEntry( const MACAddress_t * pxMACAddress,
         configASSERT( pxMACAddress != NULL );
         configASSERT( pulIPAddress != NULL );
 
+        if( ppxInterface != NULL )
```

Comment:
Not a comment about this line.
We should add the parameter description of `ppxInterface` in the Doxygen comments on line 483.